### PR TITLE
[Feature] Admin Prereg Embargo indication

### PR DESCRIPTION
**Pull request by Ghalko**
## Proposal
Adds a line to the admin prereg interface that indicates the day that the embargo is released.

## Repercussions
Other than making the interface that more crowded, and cleaning up very minor housekeeping issues, none.

### CODE REVIEW

File: admin/pre_reg/serializers.py [hunk 1]
```diff
@@ -10,8 +11,14 @@ def serialize_user(user):
...
     }
 
+
 # TODO: Write and use APIv2 serializer for this
 def serialize_draft_registration(draft, json_safe=True):
+    if draft.approval.meta['registration_choice'] == 'embargo':
+        time = parser.parse(draft.approval.meta['embargo_end_date'])
+        embargo = iso8601format(time) if json_safe else time
+    else:
+        embargo = 'Immediate'
```
Comment (sloria) [inline admin/pre_reg/serializers.py]: Avoid magic strings. This should be a named constant.

Comment (Ghalko) [inline admin/pre_reg/serializers.py]: Not quite sure where I would get this constant from. Am I to construct said constant? @sloria

Comment (sloria) [inline admin/pre_reg/serializers.py]: Yes

Comment (Ghalko) [inline admin/pre_reg/serializers.py]: Constants constructed. Resubmitted to travis.

**Comment by Ghalko** [comment]
@sloria Checks out.

**Comment by Ghalko** [comment]
Closing in favor of #4882

**Pull request closed without merging**